Makefile

```
@@ -372,6 +372,45 @@ docker-run-access:
 docker-run-ghost:
 	docker run -p 9000:9000 -p 3569:3569 "$(CONTAINER_REGISTRY)/ghost:latest" --nodeid 1234567890123456789012345678901234567890123456789012345678901234 --entries ghost-1234567890123456789012345678901234567890123456789012345678901234@localhost:3569=1000
 
+PHONY: docker-all-tools
+docker-all-tools:tool-util tool-read-badger tool-read-protocol-state tool-remove-execution-fork
+
+PHONY: docker-build-util
+docker-build-util:
+	docker build -f cmd/Dockerfile --ssh default --build-arg TARGET=util --target production \
+		-t "$(CONTAINER_REGISTRY)/util:latest" -t "$(CONTAINER_REGISTRY)/util:$(SHORT_COMMIT)" -t "$(CONTAINER_REGISTRY)/util:$(IMAGE_TAG)" .
+
+PHONY: tool-util
+tool-util: docker-build-util
+	docker container create --name util gcr.io/flow-container-registry/util:latest;docker container cp util:/bin/app ./util;docker container rm util
```

```suggestion
	docker container create --name util gcr.io/$(CONTAINER_REGISTRY)/util:latest;docker container cp util:/bin/app ./util;docker container rm util
```

Likewise for the other targets.